apps/site/site.json

```
@@ -28,21 +28,20 @@
   ],
   "websiteBanners": {
     "index": {
-      "startDate": "2025-03-17T00:00:00.000Z",
-      "endDate": "2025-04-17T00:00:00.000Z",
-      "text": "Node.js launches an official Discord community",
-      "link": "https://nodejs.org/en/blog/announcements/official-discord-launch-announcement",
-      "type": "default"
+      "startDate": "2025-04-01T00:00:00.000Z",
+      "endDate": "2025-06-01T00:00:00.000Z",
+      "text": "Node.js v18 is reaching End-Of-Life on April 30, 2025.",
```

```suggestion
      "text": "Node.js v18 is reaching End-Of-Life on April 30, 2025.",
      "link": "https://nodejs.org/en/about/previous-releases#release-schedule",
```
Up to you if you think this link acts as useful context or not